src/foptions.pas

```diff
@@ -222,7 +219,7 @@ procedure TfrmOptions.FormClose(Sender: TObject; var CloseAction: TCloseAction);
 
 procedure TfrmOptions.FormCloseQuery(Sender: TObject; var CanClose: boolean);
 begin
-  CanClose := (ModalResult in [mrOK, mrCancel]) or CycleThroughOptionEditors(False);
```

Comment:
Why do you remove `mrCancel`? DC should not ask user to save changes when user press cancel. It should ask only when user close window by "X" title button.

Reply:
for careless closing the Form.

press Cancel with quering, it seems a bit risky.

Reply:
It was such way five years ago. It is very annoying, I press `Cancel` when I want to cancel all changes. Why DC should ask me again about this?

Reply:
> It was such way five years ago. It is very annoying, I press `Cancel` when I want to cancel all changes. Why DC should ask me again about this?

I think it's normal situation if DC will asks about it too. You could by mistake press another button (.